client/src/common/components/Dropdown.jsx

```diff
@@ -94,7 +108,10 @@ Dropdown.propTypes = {
     .isRequired,
   children: PropTypes.oneOfType([PropTypes.node, PropTypes.string]).isRequired,
   dropdownClassName: PropTypes.string,
-  dropdownName: PropTypes.string
+  dropdownName: PropTypes.string,
+  location: PropTypes.shape({
+    pathname: PropTypes.string.isRequired
+  })
```

Comment:
The `location` prop is required as well (destructured without a fallback value) - please add `.isRequired`.